throttling/README.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: pattern
+title: Throttling pattern
+folder: throttling
+permalink: /patterns/throttling/
+tags:
+ - Java
+ - Difficulty-Beginner
+ - Throttling
+---
+
+## Intent
+Ensure that a given tenant is not able to access resources more than the assigned limit.
```

Comment:
Should we talk about client here instead of tenant? And service resources instead of just resources. Check for occurrences elsewhere too.